manager/controllers/utils.go

```diff
@@ -344,7 +348,13 @@ func parseTagsFromCtx(c *gin.Context, filters Filters) error {
 			LogAndRespBadRequest(c, err, err.Error())
 			return err
 		}
-		key := *tag.Namespace + "/" + tag.Key
+
+		var namespace string
+		if tag.Namespace != nil {
+			namespace = *tag.Namespace
+		}
+
+		key := namespace + "/" + tag.Key
```

Comment:
There's still a minor issue - if namespace is nil the key will start with "/".

Anyway I'm going to merge it and will correct it.